File: src/main/java/com/est/curdsample/util/TimeFormatter.java
package com.est.curdsample.util;

import java.time.LocalDate;
import java.time.LocalDateTime;
import java.time.format.DateTimeFormatter;

public class TimeFormatter {

    private static final DateTimeFormatter pattern = DateTimeFormatter.ofPattern("MM/dd/yyyy");

    public static LocalDate convertToLocalDate(String localDateStr) {
        return LocalDate.parse(localDateStr, pattern);
    }

    public static String convertToString(LocalDate localDate) {
        return localDate.format(pattern);
    }
    public static String convertToString(LocalDateTime localDateTime) {
        return localDateTime.format(pattern);
    }
}

File: src/main/java/com/est/curdsample/util/TimeFormatter.kt
package com.est.curdsample.util

import java.time.LocalDate
import java.time.LocalDateTime
import java.time.format.DateTimeFormatter

private val pattern: DateTimeFormatter = DateTimeFormatter.ofPattern("MM/dd/yyyy")

fun convertToLocalDate(localDateStr: String?): LocalDate {
    return LocalDate.parse(localDateStr, pattern)
}

fun convertToString(localDate: LocalDate): String {
    return localDate.format(pattern)
}

fun convertToString(localDateTime: LocalDateTime): String {
    return localDateTime.format(pattern)
}
